core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -408,14 +408,25 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
   final def bracket_[R1 <: R, E1 >: E]: ZIO.BracketAcquire_[R1, E1] =
     new ZIO.BracketAcquire_(self)
 
+  /**
+   * Uncurried version. Doesn't offer curried syntax and have worse
+   * type-inference characteristics, but it doesn't allocate intermediate
+   * [[scalaz.zio.ZIO.BracketAcquire_]] and [[scalaz.zio.ZIO.BracketRelease_]] objects.
+   */
+  final def bracket_[R1 <: R, E1 >: E, B](
+    release: ZIO[R1, Nothing, _],
+    use: ZIO[R1, E1, B]
+  ): ZIO[R1, E1, B] =
+    ZIO.bracket(self, (_: A) => release, (_: A) => use)
+
   /**
    * Returns an effect that, if this effect _starts_ execution, then the
    * specified `finalizer` is guaranteed to begin execution, whether this effect
    * succeeds, fails, or is interrupted.
    *
    * Finalizers offer very powerful guarantees, but they are low-level, and
    * should generally not be used for releasing resources. For higher-level
-   * logic built on `ensuring`, see [[ZIO#bracket]].
+   * logic built on `ensuring`, see `ZIO#bracket`.
```

Comment:
I'm sorry for this one, but scaladoc finds two `bracket` methods now and I cannot find method when signature contains type bounded by `type LowerR` declared inside `ZIO`.